README.md

```diff
@@ -4,3 +4,8 @@ Fellowship Companion is a web app that will help you peek into MLH Fellowship ac
 ## Local Setup
  - [Server Setup](/server/README.md)
  - [Client Setup](/client/README.md)
+ 
+## Authors
+| <img src="https://avatars2.githubusercontent.com/u/26309166?s=460&u=271b34a10104071a1af20d4e9d25f18cc2fa79d2&v=4" alt="Alberto Pérez de Rada Fiol" width="100" height="100" /> | <img src="https://avatars1.githubusercontent.com/u/28972442?s=460&u=3819b694ed41f8ebe006ce0ad8bfe9252092d72f&v=4" alt="Lakshya Khatri" width="100" height="100" /> | <img src="https://avatars1.githubusercontent.com/u/32371951?s=460&u=1dc0beea3e7a0f3ca7d0f224becd0e70886f1d41&v=4" alt="Saurabh Agarwala" width="100" height="100" /> |
```

Comment:
One cool trick, in case you didn't know: you can get the avatar of a user by appending `.png` to their profile URL, e.g. https://github.com/LakshyaKhatri.png

Reply:
wow! let's do that.